configs/components/_base-ruby-augeas.rb

```diff
@@ -3,19 +3,15 @@
 # load it with instance_eval. See ruby-x.y-augeas.rb configs.
 #
 
-pkg.add_source("file://resources/patches/augeas/ruby-augeas-0.5.0-patch_c_extension.patch")
```

Comment:
If we're not using the patches anymore, then we can probably safely delete them from the source tree as well

Reply:
makes sense